src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadSVG.java

```diff
@@ -367,7 +367,7 @@ private void doCubicCurveAbs(SVGPathSeg item, Matrix3d m) {
 		List<Point2D> points = b.generateCurvePoints(0.1);
 		for(Point2D p : points) myTurtle.moveTo(p.x,p.y);
 		pathPoint.set(p3);
-		isNewPath=true;
+		isNewPath=false;
```

Comment:
My fix for the seahorses created problems elsewhere and you intentionally corrected it, where is it that you started from a version that did not have this fix?

Reply:
I started from node-based-editor, realized I was in the wrong branch, copied the whole file, went to master, made a new branch, pasted, and that's how it got lost.  I didn't want to refactor the whole thing and I didn't mean to step on your change.  Thank goodness for git and for you!  I will change it right now.